docs/list.md

```diff
@@ -67,6 +67,14 @@ No one currently! Interested in having a prominent placement for your open sourc
 - **How to sign up**: No special signup known.
 - **Notes**: [Visit Cloudinary](https://cloudinary.com/blog/hacktoberfest-open-source-celebration-cloudinary) for the list of eligible libraries and SDKs.
 
+#### **composio**
+
+- **Swag**: Stickers, T-shirt
+- **Requirements**: Make 4 good PRs against 'hacktoberfest' topics in any of the Composio repositories.
+- **How to sign up**: Register for Hacktoberfest [here](https://hacktoberfest.com/).
+- **Issues**: Optional link to Hacktoberfest tagged issues in Composio repositories.
```

Comment:
It looks like this is incomplete. Can you please link to the `hacktoberfest` tagged issues?

Reply:
sure